asserter/construction_test.go

```diff
@@ -259,40 +250,6 @@ func TestConstructionParseResponse(t *testing.T) {
 			signed: true,
 			err:    nil,
 		},
-		"duplicate signer": {
```

Comment:
why are these tests removed?

Reply:
@shrimalmadhur My original thinking was we can remove it the error is not generated from this package, but it also doesn't hurt to include it for better testing coverage, I added it back